lib/src/base-styles/_nx-lists.scss

```diff
@@ -42,7 +41,7 @@
   }
 
   &:first-child {
-    border-top: none;
+    border-top: var(--nx-border-default);
```

Comment:
Checking whether this is redundant made me realize that our default border and subsection border variables are currently identical.  Is that right?

Reply:
> Checking whether this is redundant made me realize that our default border and subsection border variables are currently identical. Is that right?

So the subsection border in `nx-tile` does match the subsection border variable. However the lists should not have been using that variable, so I have changed it here: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/bfd417b8b2109d39bc5b389337c5f0c31970475b

Reply:
> So the subsection border in nx-tile does match the subsection border variable.

Not sure how you figure that, looking at the code it's using the variable for it.

> However the lists should not have been using that variable, so I have changed it here

What makes you say that? I seem to remember @dturner32 saying this was an example of a subsection border

Reply:
> > So the subsection border in nx-tile does match the subsection border variable.
> 
> Not sure how you figure that, looking at the code it's using the variable for it.
> 
> > However the lists should not have been using that variable, so I have changed it here
> 
> What makes you say that? I seem to remember @dturner32 saying this was an example of a subsection border

I'm looking at the colours in Sketch, and it's grey, not the blue-grey of an `nx-tile` subsection border. The top border of the list is the same blue-grey as the subsection border

Reply:
> I'm looking at the colours in Sketch, and it's grey, not the blue-grey of an nx-tile subsection border. The top border of the list is the same blue-grey as the subsection border

@dturner32 please clarify the following:
* what color "subsection borders" in general are supposed to be
* what color the tile subsection border is supposed to be
* what color the top border of a list is supposed to be
* what color the borders between rows of a list are supposed to be

Reply:
(related to the current color palette, not the new variables)
• subsection border color is #D1D6F0
• tile subsection border is also #D1D6F0 
• top border of list in #D1D6F0
• border color between rows is #E9E9E9
• yes, list borders will fall under subsection borders

Reply:
> border color between rows is #E9E9E9
> • yes, list borders will fall under subsection borders

Aren't these mutually exclusive?

Reply:
> (related to the current color palette, not the new variables)
> • subsection border color is #D1D6F0
> • tile subsection border is also #D1D6F0
> • top border of list in #D1D6F0
> • border color between rows is #E9E9E9
> • yes, list borders will fall under subsection borders

I think that it would be best if you confirmed what is in Sketch. If Sketch is incorrect then correct and post here, if Sketch is correct, post here. I don't want to move forward on this until I know that what I'm doing will match Sketch.

Reply:
I checked this against the Sketch file and these colors are coming directly from the components which are correct. That being said, these colors are different for the v2 variable updates. If those are the colors you'd rather work with I can provide those. Otherwise, the colors above match what's designed in sketch.

Reply:
> I checked this against the Sketch file and these colors are coming directly from the components which are correct. That being said, these colors are different for the v2 variable updates. If those are the colors you'd rather work with I can provide those. Otherwise, the colors above match what's designed in sketch.

I think we'll leave that for a separate ticket

Reply:
> (related to the current color palette, not the new variables)

To be clear, that is of no value since RSC is no longer in the old color palette

Reply:
> yes, list borders will fall under subsection borders

Given that you specified that there are two different colors of list borders, I have to assume this comment was only referring to one of them.  From what it sounds like, the top border of a list should be a "subsection border", that is, should match the color of tile subsections.  The borders between list items, on the other hand, should be some other kind of border that we perhaps don't have a name for.  I think I remember reading or hearing something to the effect that this latter color is also supposed to be used between table rows, is that correct?  Note that that is not currently the case - right now table borders use the "subsection" border styling.

Adding to this complication is the fact that at the moment, we have subsection borders defined equivalently to default borders.  I have to think that was a mistake at some point.

Reply:
But anyway this isn't terribly relevant to this PR.  I'm fine with straightening out the borders in another ticket so long as this PR doesn't make them worse

Reply:
> But anyway this isn't terribly relevant to this PR. I'm fine with straightening out the borders in another ticket so long as this PR doesn't make them worse

This PR does currently have it differing from master which seems like a problem

Reply:
> > But anyway this isn't terribly relevant to this PR. I'm fine with straightening out the borders in another ticket so long as this PR doesn't make them worse
> 
> This PR does currently have it differing from master which seems like a problem

Because the colour values need to be HSL I have used the v2 colours supplied by @dturner32, which is our "normal" `indigo-95` colour for the top border and `indigo-95` for the internal borders. https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/8b33df4bef0b92b8e22d0313b24a5709ecffb3e3

Also based on discussions with @rpokorny I have created  https://issues.sonatype.org/browse/RSC-614 to look at rationalizing our border colours.

Reply:
> I have used the v2 colours supplied by @dturner32, which is our "normal" indigo-95 colour for the top border and indigo-95 for the internal borders

So they're both supposed to be indigo-95? The top border on this branch is currently indigo-90

Reply:
> > I have used the v2 colours supplied by @dturner32, which is our "normal" indigo-95 colour for the top border and indigo-95 for the internal borders
> 
> So they're both supposed to be indigo-95? The top border on this branch is currently indigo-90

Crap, no that was a typo, `indigo-90` for the top border, `indigo-95` for the inside borders